code/modules/spells/spell_types/lightning.dm

```diff
@@ -65,22 +65,47 @@
 /obj/effect/proc_holder/spell/targeted/tesla/proc/Bolt(mob/origin,mob/target,bolt_energy,bounces,mob/user = usr)
 	origin.Beam(target,icon_state="lightning[rand(1,12)]",time=5)
 	var/mob/living/carbon/current = target
-	if(current.anti_magic_check())
+	var/whywouldyoueverusethisagainstajedi = FALSE
```

Comment:
this is outrageous, its unfair

Reply:
One, you're removing the magic check for some reason.
Two, that var name is ridiculous.
I'm thinking esword_counter would be good.

Reply:
@trollbreeder nah, I just moved the antimagic check to further down